lib/hanami/commands/console.rb

```diff
@@ -22,7 +22,7 @@ def reload!
         'irb'  => 'IRB'
       }.freeze
 
-      DEFAULT_ENGINE = 'irb'.freeze
+      DEFAULT_ENGINE = ['irb'].freeze
```

Comment:
irb is not required by default in code, that's why we have no 'IRB' const.
After falling back to default engine ```#load_engine``` will require it and const will be defined.